docs/components/console/manage-organization/alpha-terms.md

```diff
@@ -0,0 +1,39 @@
+---
+id: alpha-terms
+title: Alpha terms
+description: "Learn how to launch the alpha terms agreement as an admin user or owner during the pre-release phase of software development. "
+keywords: [alpha-terms, alpha, admin, pre-release]
+---
+
+:::note
+Opting in to Camunda alpha terms currently only applies to Enterprise SaaS subscriptions.
+:::
+
+An alpha test is the second phase of software testing in which a sampling of the intended audience can test the product. When Camunda refers to **alpha**, we reference anything pre-release, such as any version of the software designated as alpha, limited release, developer preview, non-production, or evaluation.
+
+Alpha terms typically refer to the specific terms and conditions that govern the use and testing of this software during its alpha phase. These terms outline the rights, responsibilities, and limitations of both the software provider (Camunda) and the users (alpha testers or early adopters) during the testing and evaluation period.
+
+Alpha terms help protect Camunda´s interests (such as protecting our intellectual property, disclaiming warranties, or limiting our liability for any issues or damages that may arise during the alpha phase), manage user expectations, encourage active participation and feedback, and ensure legal compliance during the pre-release phase of software development.
+
+:::note
+Enabling alpha is limited to [admin users and owners](/manage-users.md) of Camunda products.
+:::
+
+## Accept alpha terms
+
+To accept alpha terms for Camunda products, follow the steps below:
+
+1. Log in to Camunda Console and click the **Organization** tab to view the overview for **Organization Management**.
+2. Under the **Settings** tab, click **Opt-in** under the **Alpha features** box.
+3. Note the **Alpha Terms** modal. As the admin accepting the alpha terms, you must scroll and read through the terms prior to accepting.
+4. Once you have read the terms and scrolled through the modal, tick the box at the bottom reading **"I understand and agree to Alpha Terms"**.
+
+The system will confirm your acceptance and send a copy of the accepted alpha terms to your email address.
+
+## Utilize alpha features
+
+Once you accept the alpha terms, you can enable and disable any features you would like to use, and learn more about them.
+
+In the **Settings** tab, toggle the switch under **Status** to enable and disable the feature.
```

Comment:
Admins can also see when someone accepted the alpha terms and when features were enabled or disabled through the Activity log (see example [here](https://camunda.slack.com/archives/C0563ESV2EL/p1698143859194019))

Reply:
Added in latest commit 👍